src/main/target/OMNIBUSF4/CL_RACINGF4.mk

```diff
@@ -1,11 +1 @@
-#==CL_RACING F4 PDB rev 1.0==
-#Owner: CL_RACING desgin base on OmnibusF4 board
-#Board information:
-#- CPU - STM32F405RGT6
-#- MPU-6000
-#- Full push in SD Card Slot 
-#- Build in 5V BEC 
-#- Build in LC filter for camera and vtx
-#- Build in OSD
-#- Built in current meter, PDB
 FEATURES       += VCP SDCARD
```

Comment:
This can also go into src/main/target/OMNIBUSF4/target.mk. See src/main/target/SPRACINGF3/target.mk for how it's done.